mule-extensions-api/src/main/java/org/mule/runtime/extension/internal/loader/util/JavaParserUtils.java

```diff
@@ -243,7 +243,7 @@ public static ExternalLibraryType toMuleApi(org.mule.sdk.api.meta.ExternalLibrar
     } else if (type == org.mule.sdk.api.meta.ExternalLibraryType.NATIVE) {
       return ExternalLibraryType.NATIVE;
     } else {
-      throw new IllegalModelDefinitionException("Unsupported ExternalLIbraryType " + type);
+      throw new IllegalModelDefinitionException("Unsupported ExternalLibraryType " + type);
     }
   }
```

Comment:
are all methods in this calss still called after the move to extensions-support?

Reply:
Done.